src/compas_fab/backends/ros/backend_features/move_it_plan_motion.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from compas.utilities import await_callback
+
+from compas_fab.backends.ros.backend_features.helpers import convert_constraints_to_rosmsg
+from compas_fab.backends.ros.backend_features.helpers import convert_trajectory_points
+from compas_fab.backends.ros.backend_features.helpers import validate_response
+from compas_fab.backends.ros.messages import AttachedCollisionObject
+from compas_fab.backends.ros.messages import Header
+from compas_fab.backends.ros.messages import JointState
+from compas_fab.backends.ros.messages import MotionPlanResponse
+from compas_fab.backends.ros.messages import MotionPlanRequest
+from compas_fab.backends.ros.messages import MultiDOFJointState
+from compas_fab.backends.ros.messages import RobotState
+from compas_fab.backends.ros.messages import TrajectoryConstraints
+from compas_fab.backends.ros.planner_backend import ServiceDescription
+from compas_fab.robots import JointTrajectory
+from compas_fab.robots import Configuration
+
+
+class MoveItPlanMotion(object):
```

Comment:
These ones will inherit from the base ones, like `PlanMotion`, right?

Reply:
Right, I was going through and homogenizing one type of widget at a time.  That's the next one.